### DISCUSSION TITLE
可以优化一下折叠侧栏的那个直线/箭头么

### DISCUSSION THREAD

**Discussion by fuyn101** [SPlayer-Dev/SPlayer#100] [讨论 General]
![图片](https://github.com/imsyy/SPlayer/assets/151902446/3a9dd324-2d59-48bc-8b35-56cf4c74d9a1)

这个收起我个人感觉不是那么的额 协调 比较怪

**Comment by imsyy** [comment]
确实。可以缩小点或者去掉
